### DISCUSSION TITLE
【Rust项目发起】Rust图像识别推理Demo

### DISCUSSION THREAD

**Discussion by coderonion** [creatorsand/.github#22] [Ideas]
项目名：Rust图像识别推理Demo
项目介绍：使用rust实现了基于yolov5官方模型的图像识别推理demo，并采用slint-ui框架实现了GUI展示。
项目地址：https://github.com/sjinzh/yolov5-rs-app

**Comment by bonnie0519** [comment]
hi！想问一下社群昵称！是社群里的谁呀！

**Comment by Insomnia501** [comment]
hi，请问这个还可以加入吗？my TG：Daniel09_09

**Reply by ShirleneLiu** [reply]
@Insomnia501 @sjinzh 两位老板项目的链接贴一下
